know/rdf/management/commands/sample.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+
+class Command(BaseCommand):
+    """ Sample Django command.
+        Execute it like python manage.py sample -q query.sparql -u q:12345
+    """
+    def add_arguments(self, parser):
+        """ Adds arguments to the command """
+        parser.add_argument('--query', '-q', type=str, help="describe query")
+        parser.add_argument('--uri', '-u', type=str, help="describe uri")
+
+    def handle(self, *_, **options):

```

Comment:
I don't know what the "*" syntax is but I'll assume it's okay.

Reply:
Those are arbitrary and keywords argument fields. You can read more about them here: https://stackoverflow.com/a/36908/